sass/components/_emoticons.scss

```diff
@@ -117,6 +118,7 @@
         margin-top: 10px;
     }
 
+    /* this might be obsolete (I cannot find usages anywhere) */
```

Comment:
If there are not references in the code, can we remove it?

Reply:
probably ... the thing is: sometimes these styles are used in a very specific place somewhere in the application ... soooo ... it might not be safe to remove ... unless we are fine by adding a regression (even if it is a small one) and fix it later if it appears! :D

Reply:
Does the string `nav-tabs`appear anywhere in the code? If not, it is probably safe to remove it. This is something we have to be more careful with "composed classes" (like `item` and `item--selected`), but the name of this class seems safe to assume that if it does not exist it is not being used.

Reply:
I will remove them for now and see if regressions pop up from it to fix!